java/java-cloud-run-hello-world/Dockerfile

```diff
@@ -1,25 +0,0 @@
-# Use the official maven/Java 8 image to create a build artifact.
```

Comment:
Was the Java Dockerfile meant to be removed in this PR?

Reply:
I should move it to another PR. I need to talk to the team about mixing Dockerfiles and Jib. We should probably choose one or the other.